pkg/crc/preset/preset.go

```diff
@@ -15,16 +15,20 @@ const (
 	Microshift Preset = "microshift"
 )
 
+func presetMap() map[Preset]string {
+	return map[Preset]string{
+		Podman:     string(Podman),
+		OpenShift:  string(OpenShift),
+		OKD:        string(OKD),
+		Microshift: string(Microshift),
+	}
+}
```

Comment:
```
var presetMap = map[Preset]string {
...
}
```
should be enough, and allow us to create the map only once?

Reply:
updated.